py/selenium/webdriver/remote/webdriver.py

```diff
@@ -849,7 +849,7 @@ def get_screenshot_as_base64(self) -> str:
         """
         return self.execute(Command.SCREENSHOT)["value"]
 
-    def set_window_size(self, width, height, windowHandle: str = "current") -> None:
+    def set_window_size(self, width, height, window_handle: str = "current") -> None:
```

Comment:
This is technically not backwards compatible. Should we do the deprecation/warning/replacement process?